libraries/AP_RCProtocol/AP_RCProtocol_Backend.h

```diff
@@ -74,11 +75,53 @@ class AP_RCProtocol_Backend {
     }
     
 protected:
+    struct Channels11Bit {
+        // 176 bits of data (11 bits per channel * 16 channels) = 22 bytes.
+#if __BYTE_ORDER == __LITTLE_ENDIAN
```

Comment:
Ok - that's cool. Both STM32 and x86 are little endian so it's probably fine. But I found this great tutorial: http://mjfrazer.org/mjfrazer/bitfields/ that indicates that what I am doing is right. What I don't quite understand is that the output seems to be the reverse of what I am expecting. Anyway I will assume little endian and modify the code appropriately.

Reply:
I think you'll find that doc is right for small bitfields, not for 176 bit monsters like this one

Reply:
Yes, I think you are right. For the smaller bitfields in AP_Telem_CRSF it works as expected. So I am just going to remove the endianness here as I think it will be the same on all architectures.